### PR TITLE
UTC-252 Fix image/logo sizing issue in cards

### PR DESCRIPTION
**Pull request by UTCbridget**
# Pull Request

Closes Issue #252 Address image/logo sizing issue in cards

### CODE REVIEW

**Comment by UTCbridget** [comment]
This should fix image issues in cards.

**Comment by bmartinez287** [comment]
Any concerns with switching the cards functionality here? @brittrich

**Comment by brittrich** [comment]
I think this would work for cards where the images/cards have white backgrounds, but the impact on cards that are typical image styles might not be optimal? For example:

Screen Shot 2021-06-03 at 9 41 22 PM

I'm definitely open to whatever you all think is best, though, I'm never married to anything past-Brittany has done. ;) I do remember this being very tricky to figure out site-wide because the image sizes folks are uploading are anything but standard...maybe we add an option to the card editing interface that allows the user to choose "no crop" for the image display with a recommended image size? Choosing that option could toggle to "auto" height? Then, we introduce the new behavior as a choice folks can select if auto-cropping isn't working for them?

**Comment by bmartinez287** [comment]
Interesting, I like the idea of having both and potentially avoid radically switching current users if they are using the 100%. Do you have a URL for that image? Or maybe a before and after on how this change will affect it.
This PR got merged a bit early so we may have to undo some of it has backward compatibility issues for some items.
@brittrich

File: source/default/_patterns/00-protons/legacy/css/components/header/_utc_custom_header.css [hunk 1]
```diff
@@ -1,158 +1,191 @@
...
-    margin-left: 4px;
-  }
-  .header__toggleable-account-menu {
-    display: none;
-  }
-  .header__user-nav .js-toggle-search {
-    padding-left: 12px;
-    padding-right: 0;
-  }
+    /*Drupal 8 mega menu is no longer used. Superfish is now in play.
```
Comment (bmartinez287) [inline source/default/_patterns/00-protons/legacy/css/components/header/_utc_custom_header.css:124]: @UTCbridget let's do a pr that enables these classes again. Disabling those will break backwards compatibility if superfish does not work as expected once it reaches production. If superfish goes well we can remove them in a subsequent one.

**Comment by brittrich** [comment]
> Interesting, I like the idea of having both and potentially avoid radically switching current users if they are using the 100%. Do you have a URL for that image? Or maybe a before and after on how this change will affect it.
> This PR got merged a bit early so we may have to undo some of it has backward compatibility issues for some items.
> @brittrich

So, I just chose a random image for this card as an example, but here's a page with them in use: https://new.utc.edu/library/special-collections 

With 100%:

Screen Shot 2021-06-03 at 10 28 23 PM

With edit to auto:

Screen Shot 2021-06-03 at 10 29 11 PM

**Pull request merged**